svc/butler-srv/srv/acquire.go

```diff
@@ -0,0 +1,40 @@
+package srv
+
+import (
+	"encoding/json"
+	"github.com/Azeite-da-Quinta/jigajoga/libs/go/token"
+	"net/http"
+	"time"
+)
+
+func AcquireHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method == "POST" {
+		decoder := json.NewDecoder(r.Body)
+		var user token.Data
+		defer r.Body.Close()
+		err := decoder.Decode(&user)
+		if err != nil {
+			w.WriteHeader(http.StatusBadRequest)
+			//w.Write([]byte(err.Error()))
+		} else {
+			b, err := token.Base64ToKey(token.DefaultSecret)
+			if err != nil {
+				panic(err)
+			}
+
+			codec := token.Codec{
```

Comment:
You can create the codec once and reuse it. You can either transform this function into a method of a struct — or simply convert this function into a Closure that instantiates the codec and returns the handler function